error running blog example

I can get the other examples works but when I run `yarn dev` on the blog example, I get:
```
Running custom build: yarn run build
Usage Error: Couldn't find a script named "build".

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...

✘ [ERROR] Command failed with exit code 1: yarn run build
```

--- a/blog/wrangler.example.toml
+++ b/blog/wrangler.example.toml
@@ -6,6 +6,3 @@ compatibility_date = "2022-02-03"
 kv_namespaces = [
   { binding = "BLOG_EXAMPLE", preview_id = "", id = "" }
 ]
-
-[build]
-command = "yarn run build"

